1_processing/composition_of_compendia.ipynb

```diff
@@ -806,21 +806,1095 @@
   {
```

Comment:
<p>i just noticed that the legend colors are in the opposite order of how they occur in the plots. Can you reverse that somehow? I find when there are a lot of colors to differentiate between, it's super helpful to to have the order in the legend exactly match the order in the figure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/core-accessory-interactome/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>This is a great suggestion. I agree that this will help with readability. I'll fix this</p>